shell/common/layer_tree_holder.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_COMMON_LAYER_TREE_HOLDER_H_
+#define FLUTTER_SHELL_COMMON_LAYER_TREE_HOLDER_H_
+
+#include <memory>
+
+#include "flow/layers/layer_tree.h"
+
+namespace flutter {
+
+class LayerTreeHolder {
```

Comment:
Documentation for this class. The following would be good to answer in this docset:
1: Who owns the holder?
2: Threading requirements.
3: Which components typically push and pop a layer trees?
4: What does the new-ness of layer trees mean exactly (clarify its the target time)?

Reply:
Done